.circleci/config.yml

```diff
@@ -948,6 +950,7 @@ workflows:
           filters:
             branches:
               only:
+                - headless-staging
```

Comment:
Can we publish to a tag in npm and run e2e + graph e2e as well?

Reply:
turning on "publish_to_local_registry" for this branch will turn on all the other tests because they declare an upstream dependency on that one.

I can add publishing to a tag for this branch as well